### ISSUE TITLE
Cannot read property 'path' of undefined

### ISSUE THREAD

**Issue by sid22m** [djaodjin/djaodjin-annotate#21]
Instead of using radio button, I am using img element and angularjs, example:
```
<img class="annotate-image-select" ng-click ="changeImage()" ng-repeat = "image in imageList" value="{{image.url}}">
```
Also I changed the "change" event on annotate-image-select to "click" event in library to fire it on click instead of change.

When I want to add new image in the form of object eg 
```
{
url : "image path in form of base64 string"
}
```
 I just push it in imageList and call
```
 $('#myCanvas').annotate("push" , newImage.url);  
```
But, when I click any image. I get error:

__Cannot read property 'path' of undefined at line  228__

**Comment by StephaneRob** [comment]
Could you provide a demo (jsFiddle)?

**Comment by sid22m** [comment]
@StephaneRob Here is the fiddle: http://jsfiddle.net/te7rexkt/1/

**Comment by StephaneRob** [comment]
@sid22m I released a new version (v0.0.2).

To fix your issue:

- download latest version
- on Annotate initialisation:
```javascript
    $('#myCanvas').annotate({
        color: 'red',
        linewidth:4,
        bootstrap: true ,
        images: [$scope.imageList[0].url],
        selectEvent: "click" // this avoid to change plugin code
    });
```
- In your code replace ```value="{{image.url}}"``` by ```data-path="{{image.url}}"``` (on ```.annotate-image-select``` image element)

working fiddle : http://jsfiddle.net/te7rexkt/2/

**Comment by sid22m** [comment]
Thanks man! That was fast.

**Comment by StephaneRob** [comment]
fixed #22
